Library/Homebrew/dependency_collector.rb

```diff
@@ -57,13 +60,17 @@ def build(spec)
     parse_spec(spec, Array(tags))
   end
 
+  def glibc_dep_if_needed; end
+
+  def gcc_dep_if_needed; end
+
   def git_dep_if_needed(tags)
     return if Utils::Git.available?
 
     Dependency.new("git", tags)
   end
 
-  def brewed_curl_dep_if_needed(tags)
```

Comment:
Renamed for consistency while we're here.

Reply:
Agreed, looks good